dags/example.py

```
@@ -1,12 +1,19 @@
-    print("File has been created at /tmp/example_file.txt")
-
-def write_to_file(**kwargs):
+def create_file():
+    path = "/tmp/example_file.txt"
+    os.makedirs(os.path.dirname(path), exist_ok=True)
+    with open(path, "a", encoding="utf-8"):
+        pass
+    print(f"File has been created at {path}")
+
+def write_to_file():
+    path = "/tmp/example_file.txt"
```

Ruff (0.12.2)

Probable insecure usage of temporary file or directory: "/tmp/example_file.txt"

(S108)